fern/api-reference/op-mainnet/op-mainnet-api-faq/op-mainnet-error-codes.mdx

```diff
@@ -2,15 +2,16 @@
-| `429`      | `Optimism sequencer sender global transaction limit exceeded. This limit is set at the chain level. To ensure your request goes through, we recommend implementing retries </docs/how-to-implement-retries>`                                                           | This limit is set by Optimism at the sequencer level, see [Optimism API FAQ](/reference/optimism-api-faq). We recommend [implementing retries](/docs/how-to-implement-retries) if you have not already. If you still experience issues you can increase your throughput limit by [upgrading your account](https://dashboard.alchemy.com/settings/billing). |
-| `429`      | `Optimism sequencer global transaction limit exceeded This limit is set at the chain level. To ensure your request goes through, we recommend implementing retries </docs/how-to-implement-retries>`                                                                   | This limit is set by Optimism at the sequencer level, see [Optimism API FAQ](/reference/optimism-api-faq). We recommend [implementing retries](/docs/how-to-implement-retries) if you have not already. If you still experience issues you can increase your throughput limit by [upgrading your account](https://dashboard.alchemy.com/settings/billing). |
-| `429`      | `eth_sendRawTransaction is a method with custom rate limits that you have exceeded. If you have retries enabled, you can safely ignore this message. If not, check out </docs/reference/throughput>`                                                                        | We recommend [implementing retries](/docs/how-to-implement-retries) if you have not already. If you still experience issues you can increase your throughput limit by [upgrading your account](https://dashboard.alchemy.com/settings/billing).                                                                                                            |
+| `429`      | `Your app has exceeded its compute units per second capacity. If you have retries enabled, you can safely ignore this message. If not, check out <https://docs.alchemy.com/reference/throughput>`                                                                                              | We recommend [implementing retries](/docs/how-to-implement-retries) if you have not already. If you still experience issues you can increase your throughput limit by [upgrading your account](https://dashboard.alchemy.com/settings/billing).                                                                                                            |
```

Comment:
These aren't actually functional links - just part of the literal error messages from op-mainnet. We should leave them as is so people doing exact text search can find this page